build.sbt

```diff
@@ -1,8 +1,16 @@
+enablePlugins(GitVersioning)
+
 organization := "org.scalaz.stream"
 
 name := "scalaz-stream"
 
-version := "snapshot-0.7"
+val ReleaseTag = """^release/([\d\.]+a?)$""".r
+git.gitTagToVersionNumber := {
+  case ReleaseTag(version) => Some(version)
+  case _ => None
+}
+
+git.formattedShaVersion := git.gitHeadCommit.value      // bintray really hates -SNAPSHOT versions, so we just use the hash
```

Comment:
Does this line prevent appending "-SNAPSHOT" to the version? I think sbt-git does this only if there are uncommitted changes. So it should not be really necessary.

If it is still necessary could you change the comment to something more informative like "bintray does not allow -SNAPSHOT versions ..."?